Docs/user_guide/getting_started.rst

```diff
@@ -0,0 +1,102 @@
+
+
+What is Slicer ?
+----------------
+
+3D Slicer is:
+
+* A software platform for the analysis (including registration and interactive segmentation) and visualization (including volume rendering) of medical images and for research in image guided therapy.
+* A free, `open source <http://en.wikipedia.org/wiki/Open_source>`_ software available on multiple operating systems: Linux, MacOSX and Windows
+* Extensible, with powerful plug-in capabilities for adding algorithms and applications.
+
+Features include:
+
+* Multi organ: from head to toe.
+* Support for multi-modality imaging including, MRI, CT, US, nuclear medicine, and microscopy.
+* Bidirectional interface for devices.
+
+.. important::
+
+  There is no restriction on use, but Slicer is **NOT** approved for clinical use and intended for research. Permissions and compliance with applicable rules are the responsibility of the user. For details on the license see `here <https://www.slicer.org/wiki/License>`_.
+
+
+Hardware Requirements
+---------------------
+
+3D Slicer is an open-source package that can be used on Mac, Linux and Windows. In order to run 3D Slicer your computer must have the graphics capabilities and memory to hold the original image data and process results. A 64-bit system is required. Click here more information.
+
+
+Installing 3D Slicer
+--------------------
+
+To install Slicer, click `here <http://download.slicer.org/>`_.
+
+.. figure:: https://www.slicer.org/w/images/9/90/Screen_Shot_2016-06-01_at_1.21.15_PM.png
+
+  The Nightly version of 3D Slicer is updated nightly as groups of developers make changes. The Stable version of 3D 
+  Slicer is not updated nightly and is more rigorously tested.
+
+
+Once downloaded, follow the instructions below to complete installation.
+
+
+Further Documentation
+---------------------
+
+If you're interested in extending your knowledge, access the User Manual. See also the `archives of the users mailing list <http://slicer-users.65878.n3.nabble.com/>`_. The archive is searchable so most answers to questions can be found there.
+
+If you're a developer looking for more information, access the Developer Manual. See also `archives of the developer's mailing list <http://slicer-devel.65872.n3.nabble.com/>`_. Similar to the Users Mailing List archive, it is searchable.
+
+
+User Interface Overview
+-----------------------
+
+3D Slicer is built on a modular architecture. The Main Application GUI is divided into six components: the Application Menu Bar, the Application Toolbar, the Module GUI Panel, the Data Probe Panel, the 2D Slice Viewers, and the 3D Viewer.
+
+This section will introduce you to the basic functions on the main application's GUI. If you require detailed information, visit this page.
+
+
+Open 3D Slicer and load your own data or download sample data to explore. Go ahead and click around the user interface.
+
+
+.. image:: https://www.slicer.org/w/images/a/a6/LoadingData-16-06-06.png
```

Comment:
URL has changed: https://www.slicer.org/w/img_auth.php/a/a6/LoadingData-16-06-06.png (and probably for all similar URLs)